docs/UserGuide.md

```diff
@@ -243,12 +283,21 @@ Examples:
+| **Delete tutorial class**               | `delete_class module/<module_code> tutorial/<tutorial_class>` <br> e.g., `delete_class module/CS2103T tutorial/T09`                                                                                                                    |
+| **View all classes**                    | `list_classes`                                                                                                                                                                                                                         |
+| **Add New Student to tutorial class**   | `add_student_to_class <index/, email/ , id/> <index or email or id> module/ <module> tutorial/ <class>` <br> e.g., `add_student_to_class index/1 module/CS2103T tutorial/T09`                                                          |
+| **Delete students from tutorial class** | `delete_student_from_class email/<email> module/ <module> tutorial/ <class>` or `delete_student_from_class id/<id> module/ <module> tutorial/ <class>` <br> e.g., `delete_student_from_class id/A1234561Z module/CS2103T tutorial/T09` |
+| **List students of a tutorial class**   | `class_list_student module/ <module> tutorial/ <class>` <br> e.g., `class_list_students module/CS2103T tutorial/T09`                                                                                                                   |
+| **Delete module**                       | `delete_module module/<module_code>` <br> e.g., `delete_class module/CS2103T`                                                                                                                                                          |
+| **Add new team**                        | `add_team module/<moduleCode> tutorial/<tutorialClass> name/<team_name> size/<team_size>` <br> e.g., `add_team module/CS2103T tutorial/T09 name/Team 1 size/5`                                                                         |
+| **Delete team**                         | `delete_team module/<moduleCode> tutorial/<tutorialClass> name/<team_name>` <br> e.g., `delete_team module/CS2103 tutorial/T09 name/Team 4`                                                                                            |
+| **View team**                           | `view_teams <index/, name/> <index or name> module/<moduleCode> tutorial/<tutorialClass>` <br> e.g., `view_teams name/Team 1 module/CS2103T tutorial/T09`                                                                              |
+| **Randomly allocate team**              | `/random_teams module/<module> tutorial/<tutorialClass> teams/<number_of_teams>` <br> e.g., `random_teams module/CS2103 tutorial/T09 teams/4`                                                                                          |
```

Comment:
Theres also allocate student to team and delete student from team commands missing